imgui-core/src/main/kotlin/imgui/demo/showDemoWindowMisc.kt

```diff
@@ -130,8 +130,11 @@ object showDemoWindowMisc {
                 text("Keys mods: $ctrl$shift$alt$super_")
                 text("Chars queue:")
                 io.inputQueueCharacters.forEach { c ->
-                    sameLine();  text("\'%c\' (0x%04X)", if(c > ' ' && c.i <= 255) c else '?', c)
-                } // FIXME: We should convert 'c' to UTF-8 here but the functions are not public.
+                    // UTF-8 will represent some characters using multiple bytes, so we join them here
+                    // example: 'ç' becomes "0xC3, 0xA7"
+                    val bytes = c.toString().toByteArray(Charsets.UTF_8).map { String.format("0x%X", it) }.joinToString()
```

Comment:
Ps: this might be simplified to 
`c.toString().toByteArray().joinToString { "0x%X".format(it) }`

Reply:
Nice catch!